packages/transactions/src/builders.ts

```diff
@@ -1273,6 +1273,8 @@ export interface ReadOnlyFunctionOptions {
   network?: StacksNetworkName | StacksNetwork;
   /** address of the sender */
   senderAddress: string;
+  /** tip height **/
+  tip?: number;
```

Comment:
Type of `number` does not match the RPC docs and afaik won't work and could potentially cause problems.